tests/test_bad_seccomp_filter_length

```diff
@@ -2,11 +2,11 @@
 
 set -e
 
-. "$(pwd)/common.sh"
+. "$(pwd)"/common.sh
```

Comment:
No. It was a cleanup after a sed to fix some shellcheck stuff and I left this change for consistency in other tests.